catalogs/sources/gtfs/schedule/us-california-blue-lake-rancheria-transit-system-gtfs-598.json

```diff
@@ -1,21 +0,0 @@
-{
-    "mdb_source_id": 598,
```

Comment:
In order to preserve the Mobility Database IDs, we can’t delete sources. This issue is here so we can add [deprecation info](https://github.com/MobilityData/mobility-database-catalogs/issues/131) to the schema.

Reply:
This is unfortunate as I don't think the definition of what a unique ID even represents has been defined (see comment in I believe #36 ) - so it seems impractical /impossible to "hold them constant" if we don't know what they represent.  

I would suggest we don't "hold anything to be frozen/backwards compatible until this issue is resolved/closed as it will end up in a LOT of messiness.

Reply:
I backed out this change in the PR.